### DISCUSSION TITLE
How Can I Run npm package For My Website On My computer

### DISCUSSION THREAD

**Discussion by enjoyincity** [npm/feedback#91] [Website] [answered]
Dear Sir / Madam,
I am an ordinary website designer and i only can edit in html source code. I have noticed that in premade website template specially bootstrap based template, JavaScript is not complied and minified. I would like to know how can i install and run npm on my local computer for rectifying the errors on my website [](https://www.enjoyindehradun.com)

**Comment by ethomson** [comment, marked as answer]
Hello @enjoyincity!  I'm sorry that you've run into difficulty, but I'm afraid that this feedback forum is for product improvement feedback for the npm family of products.  For assistance, I would encourage you to ask on [Stack Overflow](https://stackoverflow.com).

**Reply by enjoyincity** [reply]
Thanks and i will try to ask my problem on stack overflow
